### PR TITLE
Fix formatting in setup.py and bump version to 0.3.0.

### PR DESCRIPTION
**Pull request by b4hand**
The plan is to tag this release to address the concerns in #32.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -27,33 +27,35 @@
...
-    author           = 'Dan Lecocq',
-    author_email     = 'dan@moz.com',
-    classifiers      = [
+setup(
+    name='simhash',
+    version='0.3.0',
+    description='Near-Duplicate Detection with Simhash',
+    url='http://github.com/seomoz/simhash-py',
+    author='Dan Lecocq',
+    author_email='dan@moz.com',
```
Comment (sistawendy) [inline setup.py]: Is this correct?

**Comment by sistawendy** [comment]
One general thing: did you check the formatting with `pylint` or some other linting tool?

**Comment by b4hand** [comment]
pylint

File: setup.py (hunk 1 above)
Comment (b4hand) [inline setup.py]: It's unclear who should be the author at this point. I'm also not sure it really matters. I didn't feel the need to change it at the moment.

Honestly, I'm not sure what the best thing should be for public repos, we probably don't want to have an individual email, but we also don't want to publish internal mailing list names, so it's kind of a toss up.

**Pull request merged**